docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -150,7 +150,79 @@ You should see the URL for your bootstrap package as the value for `mdm.macos_se
+
+2. Open the automatic enrollment profile and replace the `profile_name` key with your organization's name.
+
+3. View the the list of macOS Setup Assistant properties (panes) [here in Apple's Device Management documentation](https://developer.apple.com/documentation/devicemanagement/skipkeys) and choose which panes to hide from your end users.
+
+4. In your automatic enrollment profile, edit the `skip_setup_items` array so that it includes the panes you want to hide.
+
+### Step 2: upload the profile to Fleet
+
+Choose which team you want to add the automatic enrollment profile to.
+
+In this example, we'll add the automatic enrollment profile to the "Workstations (canary)" team. Only hosts that automatically enroll to this team will see the custom macOS Setup Assistant.
+
+2. Create a `workstations-canary-config.yaml` file:
+
+```yaml
+apiVersion: v1
+kind: team
+spec:
+  team:
+    name: Workstations (canary)
+    mdm:
+      macos_setup:
+        macos_setup_assistant: ./path/to/automatic_enrollment_profile.json
+    ...
+```
+
+Learn more about team configurations options [here](./configuration-files/README.md#teams).
+
+To customize the macOS Setup Assistant for hosts that automatically enroll to "No team," we'll need to create an `fleet-config.yaml` file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_setup:
+      macos_setup_assistant: ./path/to/automatic_enrollment_profile.json
+  ...
+```
+
+Learn more about configuration options for hosts that aren't assigned to a team [here](./configuration-files/README.md#organization-settings).
+
+3. Add an `mdm.macos_setup.macos_setup_assistant` key to your YAML document. This key accepts a path to your automatic enrollment profile.
+
+4. Run the `fleetctl apply -f workstations-canary-config.yml` command to upload the automatic enrollment profile to Fleet.
+
+### Step 3: test the custom macOS Setup Assistant
+
+Testing requires a test Mac that is present in your Apple Business Manager (ABM) account. We will wipe this Mac and use it to test the custom macOS Setup Assistant.
+
+1. Wipe the test Mac by selecting the Apple icon in top left corner of the screen, selecting **System Settings** or **System Preference**, and searching for "Erase all content and settings." Select **Erase All Content and Settings**.
+
+2. In Fleet, navigate to the Hosts page and find your Mac. Make sure that the host's **MDM status** is set to "Pending."
+
+> New Macs purchased through Apple Business Manager appear in Fleet with MDM status set to "Pending." Learn more about these hosts [here](./MDM-setup.md#pending-hosts).
+
+3. Transfer this host to the "Workstations (canary)" team by selecting the checkbox to the left of the host and selecting **Transfer** at the top of the table. In the modal, choose the Workstations (canary) team and select **Transfer**.
```

Comment:
Rather than assuming a "workstations canary" team, can this be more general? Or maybe set it up a bit more. "Let's assume you are using a team called..." or something like that. The "Workstations (canary)" team comes out of nowhere.

Reply:
Good point. I agree this could use more setup. I updated the section here: https://github.com/fleetdm/fleet/pull/12812/files#diff-53f79a9bcbecfe28f586509c8c8012fc032b247860f51dcbc0c0254d1644a0b2R179-R181

If this looks good, I'll make the same change to other areas of the docs where we use the workstations canary team. For example profiles: https://fleetdm.com/docs/using-fleet/mdm-custom-macos-settings#step-2-upload-configuration-profile-to-fleet